travel-event-host-app/src/lib/mongodb.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const { MONGODB_URI } = process.env;
+
+if (!MONGODB_URI) {
+  throw new Error("Invalid enviroment variable: MONGODB_URI");
+}
+
+export const connectMongoDB = async () => {
+  try {
+    const { connection } = await mongoose.connect(MONGODB_URI);
+
+    if (connection.readyState === 1) {
+      return Promise.resolve(true);
```

Comment:
Could we simply return 'true' since the async function is already going to return a promise?